package.json

```diff
@@ -32,7 +32,7 @@
       "require": "./dist/index.js",
       "import": "./dist/index.mjs"
     },
-    "./module": "./dist/module/index.mjs"
+    "./error": "./dist/error.mjs"
```

Comment:
```suggestion
```
I don't think this makes sense. There should be only one point of export in this package.

Reply:
(in other words, 1 way to import things from this package)

Reply:
This is here in this, as in all of our libraries, as a way to import directly `import x from "@inrupt/errors/filename"` when using ES6 modules.

Reply:
I think we need to do this as an object, so the typescript types work; there's a issue on github on another repo about this, I think solid-client-js

Reply:
Yeah, you'd need a typesVersions field (like [the one I added to solid-client-access-grant-js](https://github.com/inrupt/solid-client-access-grants-js/pull/17/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)).

But for such a small package, it doesn't make sense to enable a sub-module import syntax.

It might actually even be a bad idea for most larger libraries (cost/benefits) but that's not the discussion at hand here.

Reply:
Also, more to the point, there is only 1 "module" so really here, there is literally zero advantage.

Reply:
Agree for this library sub-module imports probably doesn't make sense.

Reply:
I think it's important to maintain consistency with our other libraries - I don't see any significant "cost" to keeping this here.

Reply:
I think the kind of consistency we should work towards is applying a set of best practices in coding, one of which would be not to expose sub-modules when a package doesn't have any.

I also think that sub-modules only make sense in specific situations and I'm not sure they're curently warranted in any of our packages since they increase complexity and provide little benefit unless I'm missing something.

Reply:
> I also think that sub-modules only make sense in specific situations and I'm not sure they're curently warranted in any of our packages since they increase complexity and provide little benefit unless I'm missing something.

Which is fine, but that's not a discussion for this PR. That is a separate discussion we can have as it affects all of our libraries.

Reply:
OK, let's schedule that discussion soon 🤞

(also because as it stands, our submodule exports are broken as missing types)